acceptance-tests/run-local.sh

```diff
@@ -76,14 +85,17 @@ if [ "$(uname)" == "Darwin" ]; then
     docker_mac_check_cgroupsv1
 fi
 
-# Build acceptance test image
-pushd "$SCRIPT_DIR/../ci" || exit 1
- docker build -t haproxy-boshrelease-testflight .
-popd || exit 1
+build_image "${REPO_DIR}/ci"
+prepare_docker_scratch
 
 # Run acceptance tests
-if [ -n "$FOCUS" ]; then
-  docker run --privileged -v "$REPO_DIR":/repo -e REPO_ROOT=/repo -e FOCUS="$FOCUS" haproxy-boshrelease-testflight bash -c "cd /repo/ci/scripts && ./acceptance-tests ; sleep infinity"
+if [ -n "$KEEP_RUNNING" ] ; then
+  echo
+  echo "*** KEEP_RUNNING enabled. Please clean up docker scratch after removing containers: ${DOCKER_SCRATCH}"
+  echo
+  docker run --privileged -v "$REPO_DIR":/repo -v "${DOCKER_SCRATCH}":/scratch/docker -e REPO_ROOT=/repo -e FOCUS="$FOCUS" -e KEEP_RUNNING="${KEEP_RUNNING}" haproxy-boshrelease-testflight bash -c "cd /repo/ci/scripts && ./acceptance-tests ; sleep infinity"
 else
-  docker run --rm --privileged -v "$REPO_DIR":/repo -e REPO_ROOT=/repo haproxy-boshrelease-testflight bash -c "cd /repo/ci/scripts && ./acceptance-tests"
+  docker run --rm --privileged -v "$REPO_DIR":/repo -v "${DOCKER_SCRATCH}":/scratch/docker -e REPO_ROOT=/repo -e KEEP_RUNNING="" haproxy-boshrelease-testflight bash -c "cd /repo/ci/scripts && ./acceptance-tests"
+  echo "Cleaning up docker scratch: ${DOCKER_SCRATCH}"
+  sudo rm -rf "${DOCKER_SCRATCH}"
```

Comment:
would it be possible to do this with an `EXIT` trap in the other case?

Reply:
I think that already has `service docker stop` mapped. It is possible to map multiple things to a trap, but it gets complicated. So far this works.

Reply:
My comment was (somewhat intentional, but confusingly) on the wrong line. You remove the scratch folder here for the "non-KEEP_RUNNING" part. But for the `KEEP_RUNNING` it's just a note in the README that might be overlooked.

Reply:
It also prints message in terminal. I have now also adjusted cleanup script to remove any orphaned scratch older than 2w.

Reply:
great!